Add known issue caused by credentials error with S3+SQS (#2168)
elastic/observability-docs: docs/en/ingest-management/release-notes/release-notes-8.4.asciidoc

[[release-notes-8.4.1]]
== {fleet} and {agent} 8.4.1

[discrete]
[[known-issues-8.4.1]]
=== Known issues

// tag::credentials-error[]
.Credentials error prevents {agent} from ingesting AWS logs
[%collapsible]
====

*Details*

> {agent}s configured to use the AWS S3 and SQS integrations may return the

Reviewer: I would be "less" specific: meaning is a problem related to any AWS integration and not specific to S3 and SQS.
It also involves Crowdstrike, Cisco Umbrella and AWS Logs integrations.

> {agent}s configured to use the AWS S3 and SQS integrations may return the

Author: ```suggestion
{agent}s configured to use the AWS integration may return the
```

following error:

[source,shell]
----

> sqs ReceiveMessage failed: operation error SQS: ReceiveMessage, https response

Reviewer: beware that the error message might differ for some users: it's anyway most of the time related to authentication and/or region

error StatusCode: 403, RequestID: cb57783a-505f-5099-9160-23b8eea8ddbb,
api error SignatureDoesNotMatch: Credential should be scoped to a valid region. 
----

This error was introduced by a breaking change in the AWS library.

*Impact* +

{agent} is unable to ingest AWS logs and some metrics. To resolve this error:

> * If you are using the default domain `amazonaws.com`, upgrade the AWS

Author: Do we need to mention explicitly that the custom endpoint configuration must be removed, or will the package upgrade just remove it?

Reviewer 2: The package upgrade will remove it 🙂

integration package to version 1.23.4 to apply the temporary fix added in

> https://github.com/elastic/integrations/pull/4103[PR #4103].

Reviewer: We can add that is this does not still solve the problem they have to set an AWS region (either from env variable, credentials or instance profile) where the Agent is running.

> * If you are using the default domain `amazonaws.com`, upgrade the AWS
> integration package to version 1.23.4 to apply the temporary fix added in
> https://github.com/elastic/integrations/pull/4103[PR #4103].

Author: ```suggestion
* If you are using the default domain `amazonaws.com`, upgrade the AWS
integration package to version 1.23.4 to apply the temporary fix added in
https://github.com/elastic/integrations/pull/4103[PR #4103]. If this does not
solve your problem, set the AWS region (either from an environment variable,
credentials or instance profile) where {agent} is running.
```

[[release-notes-8.4.1]]
== {fleet} and {agent} 8.4.1

[discrete]
[[known-issues-8.4.1]]
=== Known issues

// tag::credentials-error[]
.Credentials error prevents {agent} from ingesting AWS logs
[%collapsible]
====

*Details*

{agent}s configured to use the AWS integration may return an error similar to the following:

> following error:

Author: ```suggestion
```
